http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpFullResponse.java

```diff
@@ -166,5 +167,8 @@ default Builder putHeader(String headerName, String headerValue) {
          * This is usually done by closing the service connection.</p>
          */
         Builder content(AbortableInputStream content);
+
```

Comment:
Are all the methods on this builder still needed?

Reply:
Yes as discussed.